geoindicators/src/test/groovy/org/orbisgis/geoclimate/geoindicators/GridIndicatorsTests.groovy

```diff
@@ -0,0 +1,125 @@
+package org.orbisgis.geoclimate.geoindicators
+
+import org.junit.jupiter.api.BeforeAll
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import org.orbisgis.data.H2GIS
+import org.orbisgis.data.POSTGIS
+import org.orbisgis.geoclimate.Geoindicators
+
+import static org.junit.jupiter.api.Assertions.assertEquals
+import static org.junit.jupiter.api.Assertions.assertTrue
+import static org.orbisgis.data.H2GIS.open
+
+class GridIndicatorsTests {
+
+    @TempDir
+    static File folder
+
+    private static H2GIS h2GIS
+
+    @BeforeAll
+    static void beforeAll() {
+        h2GIS = open(folder.getAbsolutePath() + File.separator + "gridIndicatorsTests")
+    }
+
+    @BeforeEach
+    void beforeEach() {
+    }
+
+    @Test
+    void multiscaleLCZGridTest() {
+        //Data for test
+        h2GIS.execute("""
+        --Grid values
```

Comment:
Not sure but is this test have an equal number of LCZ for one situation and thus has to use the weights to check that the weigths are OK ?

Reply:
Weights are always used